INSTALL.md

```diff
@@ -392,6 +392,104 @@ Launch XAMPP Control Panel and navigate to LibreEHR Setup in your browser: `\Lib
 
  **Once you see the screen above, you have successfully setup LibreEHR! Congratulations!**
 
+## On Mac With MAMP (For Local Testing)
+Apache can be used on Mac to install LibreHealth. However, if you have installation problems, you can use MAMP.
+
+#### Setup
+Install MAMP version 3.5.6 or under (Newer versions come with PHP 7.1 which is not compatible with LibreHealthEHR. If youdo install a newer version, make sure to use PHP versions under 7.1). You can find older versions.
+
+#### Note: PHP must not be version 7.1 or above. Version 7.0 and 5.6 can both be used.
+
+Firstly, download [MAMP](https://mamp.info/en/) version 3.5.6. Currently, we do not support PHP versions 7.1. MAMP 3.5.6 comes with PHP 7.0 so you can use it without much configuration. Older Versions (including version 3.5.6) can be found [here](https://mamp.info/en/downloads/older-versions/)
+
+After Installation, the directory MAMP can be found in Applications. Go to Applications > MAMP and launch MAMP.app .
+
+ ![MAMPControl](./Documentation/1_Installing/images/MAMP/MAMP_Control.png)
+
+ Now navigate to Preferences > Ports and change your Apache Port to 80 and MySQL port to 3306. This can be done by pressing 'Set Web and MySQL ports to 80 & 3306'.
+
+ ![MAMPSettings](./Documentation/1_Installing/images/MAMP_installation/Settings.png)
+
+ Finally, you should make sure that you have got the right PHP version.
+
+ ![MAMPPHP](./Documentation/1_Installing/images/MAMP_installation/PHP.png)
+
+ Version 5.6.10 is recommended but any version which is not 7.1 and newer is fine.
+
+2. To make sure the LibreHealthEHR functions properly, you will need to make some changes to your php.ini file. Different versions of php have different php.ini files.
+For PHP 7.0.8
+`/Applications/MAMP/conf/php7.0.8/php.ini`
+For PHP 5.6.10
+`/Applications/MAMP/conf/php5.6.10/php.ini`
+
```

Comment:
Ensure that you are not advocating for people to use a version that will be deprecated and unsupported in a month and change.

Reply:
Yeah, actually no 7.0 or earlier should be used.  There are issues.  Might as well drop them from your instructions, or at least note the "minimum version" and "configurator beware"